.github/workflows/security-scans.yml

```diff
@@ -0,0 +1,226 @@
+          output_formats: json,sarif
+
+      - name: Upload KICS results
+        if: always() && matrix.tool == 'kics'
+        uses: actions/upload-artifact@v3
+        with:
+          name: kics-results
+          path: results.json
+
+      - name: Kubeconform Scan
+        if: matrix.tool == 'kubeconform'
+        run: |
+          echo "::group::Kubeconform installation"
+          go install github.com/yannh/kubeconform/cmd/kubeconform@${{ matrix.version }}
+          echo "::endgroup::"
+          
+          cd operator/config/default
+          sed -i 's/\- ..\/crd//' kustomization.yaml && kustomize build ./ > /tmp/operator-manifest.yaml
+          cd ../crd && kustomize build ./ > /tmp/crds.yaml
+          cd ../../../scheduler/manifests/install && kustomize build ./ > /tmp/scheduler-manifest.yaml
+
+          curl -s https://raw.githubusercontent.com/yannh/kubeconform/${{ matrix.version }}/scripts/openapi2jsonschema.py > /tmp/openapi2jsonschema.py
+          mkdir -p /tmp/schema && cd /tmp/schema
+          python3 ../openapi2jsonschema.py ../crds.yaml
+          cd .. && echo "---" >> operator-manifest.yaml && cat operator-manifest.yaml scheduler-manifest.yaml > manifest.yaml
+          kubeconform -schema-location default -schema-location './schema/{{ .ResourceKind }}_{{ .ResourceAPIVersion }}.json' -schema-location 'https://raw.githubusercontent.com/datreeio/CRDs-catalog/main/cert-manager.io/{{.ResourceKind}}_{{.ResourceAPIVersion}}.json' -summary manifest.yaml
+
+      - name: Kubescape Scan
+        if: matrix.tool == 'kubescape'
+        env:
+          FAILURE_PERCENTAGE: 10
+        run: |
+          echo "::group::Kubescape installation"
+          curl -s https://raw.githubusercontent.com/armosec/kubescape/master/install.sh | /bin/bash
+          echo "::endgroup::"
+          # Docs on how to configure exceptions: https://hub.armosec.io/docs/exceptions
+          kubescape scan framework ${{ matrix.kubescape-framework }} -v -t ${{ env.FAILURE_PERCENTAGE }} --exceptions ./.github/.kubescape/exceptions.json --controls-config ./.github/.kubescape/controls-inputs.json scans --enable-color
+          
+  trivy:
+    name: Trivy
+    runs-on: ubuntu-22.04
+    needs: prepare-security-scans
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - "functions-runtime"
+          - "keptn-lifecycle-operator"
+          - "scheduler"
+    steps:
+      - name: Download images
+        id: download_images
+        uses: actions/download-artifact@v3
+        with:
+          name: images
+          path: images
+
+      - name: Trivy image scan
+        uses: aquasecurity/trivy-action@0.8.0
+        with:
+          input: "images/${{ matrix.image }}-image.tar/${{ matrix.image }}-image.tar"
+          severity: 'CRITICAL,HIGH'
+          exit-code: '1'
+
+  govulncheck:
+    name: Govulncheck
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        artifacts:
```

Comment:
[nitpick]: i think it would make more sense to have this in singular: `artifact`